package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "private": false,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start"
+  },
+  "dependencies": {
```

Comment:
We don't know what dependences do we realy need for the project, let's create fully blank project with react only

Reply:
Information on dependencies is being formed now, so I left the data empty #2